cypress/fixtures/team.json

```diff
@@ -33,6 +33,6 @@
   "created": "2023-02-10T15:08:32.547728Z",
   "modified": "2023-02-10T15:08:32.547752Z",
   "name": "Team 2 Org 0",
-  "description": "",
+  "description": "test",
```

Comment:
I like to be a bit more verbose with the values here so it is clear what is being rendered. I often use This is a description. for this value. Could make trouble shooting a bit easier in the future.